cdash/CTestTestfile.cmake

```diff
@@ -1,7 +1,9 @@
-add_test(build-tests "make" "-C" "../tmp" "-j4" "all-test-tests" "all-test-examples" "all-test-benchmarks" "OPTS=\"-g\"")
+# Disabled since that won't work in the cmake build:
+# add_test(build-tests "make" "-C" "../tmp" "-j4" "all-test-tests" "all-test-examples" "all-test-benchmarks" "OPTS=\"-g\"")
```

Comment:
Would it work to add build-tests, build-examples, and build-benchmarks rules that can pass -j4 to keep build time down?

Reply:
Yes, we can think about something like that. For the autobuild, this isn't very critical, since it is not time constrained.

Reply:
Wouldn't it be important for machines with finite allocations of core-hours?

Reply:
Maybe, but the total build time for a sequential build of tests/ examples/ and benchmarks/ is less than 10 mins from what I've seen. In any case, this won't matter for now, since we run `./build all-test` for these builds anyways.

Reply:
I restored these by splitting up the building of tests into separate tests.